lexer/lexer.go

```diff
@@ -212,7 +212,12 @@ func (lex *Lexer) readFixToken() token.Token {
 	}
 
 	lex.nextChar()
```

Comment:
read nextChar just after creating the lexeme. Otherwise, the fix token has the wrong character